src/NuGet.Core/NuGet.Protocol/Plugins/MessageMethod.cs

```diff
@@ -91,6 +91,11 @@ public enum MessageMethod
         /// <summary>
         /// Set log level
         /// </summary>
-        SetLogLevel
+        SetLogLevel,
+
+        /// <summary>
+        /// Get operation claims
+        /// </summary>
+        GetAuthCredentials,
```

Comment:
I'd prefer that you not abbreviate here, especially since it could be authentication or authorization.